test/domains/vass/domain-delete.chpl

```diff
@@ -0,0 +1,4 @@
+// formerly BenA's types/records/compilerErrors/deleteRecordMessage
```

Comment:
I disagree with moving this test to a user-named path (` test/arrays/vass`). I stand by my philosophy that new tests should be put in descriptive directories, such as `test/types/records/compilerErrors/`. If you feel strongly that it should be described as an `array` or `domain` test rather than a `record` test, then I would at least put it in something like:

- `test/domains/compilerErrors/`
- `test/arrays/compilerErrors/`